stable/datadog/values.yaml

```diff
@@ -184,6 +211,12 @@ datadog:
     #
     enabled: false
 
+    ## @param port - integer - optional - default: 8126
+    ## Override the trace Agent DogStatsD port.
+    ## Note: Make sure your client is sending to the same UDP port.
+    #
+    # port:
```

Comment:
```suggestion
    # port: 8126
```
?

Reply:
it is 2 way of doing it. 
I think it makes sens to default it in the `values.yaml`, it simplify the `{{ default 8126 .Values.datadog.apm.port }}` to `{{ .Values.datadog.apm.port }}`